atomsci/ddm/pipeline/compare_models.py

```diff
@@ -1487,7 +1484,7 @@ def get_summary_metadata_table(uuids, collections=None):
                          'Split UUID':    split_uuid,
                          'Dataset Key':   data_params['dataset_key']}
             else:
-                architecture = 'unknown'
+                _architecture = 'unknown'
```

Comment:
1487: architecture='unknown' is never added to list, which might result in mismatches unless the model metadata is merged by "Name" or something